docs/event-bus/docs/03-03-service-programming-model.md

```diff
@@ -27,9 +27,7 @@ The following HTTP Headers provide information about the Event Metadata.
 | **kyma-event-time** | Business Event's time delivered to the microservice. |
 | **kyma-event-type** | Business Event's type delivered to the microservice. |
 | **kyma-event-type-version** | Business Event's version delivered to the microservice. |
-| **kyma-source-environment** | Business Event's source Namespace delivered to the microservice. |
-| **kyma-source-namespace** | Business Event's source Namespace delivered to the microservice. |
-| **kyma-source-type** | Business Event's source type delivered to the microservice. |
+| **kyma-source-id** | Identifies the origin of events. This can be an external solution or an defined identifier for internally generated events. |
```

Comment:
```suggestion
| **kyma-source-id** | Identifies the origin of events. This can be an external solution or a defined identifier for internally generated events. |
```